packages/bpk-component-checkbox/README.md

```diff
@@ -26,15 +26,20 @@ export default () => (
 
 ## Props
 
-| Property   | PropType | Required | Default Value |
-| ---------- | -------- | -------- | ------------- |
-| name       | string   | true     | -             |
-| label      | node     | true     | -             |
-| required   | bool     | false    | false         |
-| disabled   | bool     | false    | false         |
-| white      | bool     | false    | false         |
-| smallLabel | bool     | false    | false         |
-| valid      | bool     | false    | null          |
+| Property      | PropType | Required | Default Value |
+| ------------- | -------- | -------- | ------------- |
+| name          | string   | true     | -             |
+| label         | node     | true     | -             |
+| required      | bool     | false    | false         |
+| disabled      | bool     | false    | false         |
+| white         | bool     | false    | false         |
+| smallLabel    | bool     | false    | false         |
+| valid         | bool     | false    | null          |
+| indeterminate | bool     | false    | false         |
+
+## `indeterminate` prop
+
+This property is used to set the state of the checkbox to `indeterminate` which displays with a minus sign. This is used when there is checkbox group and the parent displays this state when not all children checkboxes are selected.
```

Comment:
What happens if the checkbox is `checked` and `indeterminate`?
We should clearly document which takes precidence.

In fact, we should make state an enum so that it's `empty`, `checked` or `indeterminate` but that would be heavily breaking

Reply:
So it uses psudeo classes so can't make state an enum as its just properties which applies the styles. 

I will check to see what happens with precedence regarding this web standard and how it works, as it will be out of our control

Reply:
Resolved by adding more information based on how the standard and functionality works